<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MININT-EUM7PT4" ignored="0" name="Test" tests="12" failures="2" timestamp="2021-04-12T13:39:38 IST" time="0.082" errors="0">
  <testcase name="credentials" time="0.015" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials -->
  <testcase name="credentials" time="0.0" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials -->
  <testcase name="credentials" time="0.001" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials -->
  <testcase name="credentials" time="0.001" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials -->
  <testcase name="credentials" time="0.001" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials -->
  <testcase name="credentials" time="0.001" classname="com.TestNG.My_Test">
    <failure type="java.lang.AssertionError" message="expected [Keerthi] but found [Dharshini]">
      <![CDATA[java.lang.AssertionError: expected [Keerthi] but found [Dharshini]
at com.TestNG.My_Test.credentials(My_Test.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- credentials -->
  <testcase name="credentials1" time="0.001" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials1 -->
  <testcase name="credentials1" time="0.001" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials1 -->
  <testcase name="credentials1" time="0.001" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials1 -->
  <testcase name="credentials1" time="0.001" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials1 -->
  <testcase name="credentials1" time="0.001" classname="com.TestNG.My_Test">
    <skipped/>
  </testcase> <!-- credentials1 -->
  <testcase name="credentials1" time="0.001" classname="com.TestNG.My_Test">
    <failure type="java.lang.AssertionError" message="expected [Keerthi08] but found [Dharshini15]">
      <![CDATA[java.lang.AssertionError: expected [Keerthi08] but found [Dharshini15]
at com.TestNG.My_Test.credentials1(My_Test.java:26)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- credentials1 -->
</testsuite> <!-- Test -->
